webapp/channels/src/components/emoji_picker/emoji_picker.tsx

```diff
@@ -159,6 +160,23 @@ const EmojiPicker = ({
         return emoji;
     };
 
+    const selectFirstEmoji = (emojiPositions: EmojiPosition[]) => {
+        const firstEmoji = emojiPositions[0];
+        if (!firstEmoji) {
```

Comment:
We can just check falsy on `emojiPositions[0]` to simplify
```suggestion
        if (!emojiPositions[0]) {
```